cuda/test/matrix/hybrid_kernels.cpp

```diff
@@ -79,12 +79,13 @@ class Hybrid : public ::testing::Test {
             std::normal_distribution<>(-1.0, 1.0), rand_engine, ref);
     }
 
-    void set_up_apply_data(int num_stored_elements_per_row = 0, int stride = 0)
+    void set_up_apply_data(int num_stored_elements_per_row = 0, int stride = 0,
```

Comment:
Are these parameters even needed (`num_stored_elements_per_row` and `stride`)?
I think we can delete them since we don't use them below in the function.

Reply:
Yes, I will delete them in the next PR